dev/docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+  nucypher-dev:
+    volumes:
+      - ../..:/code
```

Comment:
Does this line make this docker-compose file `pwd` dependent?

Reply:
nope